ReactWindows/ReactNative.Bridge/RemoteDebuggerJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace ReactNative.Bridge
+{
+    public class RemoteDebuggerJavaScriptBundleLoader : IJavaScriptBundleLoader
```

Comment:
>RemoteDebuggerJavaScriptBundleLoader [](start = 17, length = 36)

Should we be more consistent about the categories of things that are either in or out of the PCL?  E.g., if one IJavaScriptBundleLoader implementation is out of PCL, then maybe all should be out (e.g., it would be expected that each platform implement all of its bundle loaders...)